packages/@orbit/jsonapi/src/lib/operations/transform-operations.ts

```diff
@@ -0,0 +1,187 @@
+import {
+  buildTransform,
+  TransformOrOperations,
+  Operation,
+  Transform,
+  AddRecordOperation,
+  RemoveRecordOperation,
+  AddToRelatedRecordsOperation,
+  RemoveFromRelatedRecordsOperation,
+  ReplaceRelatedRecordOperation,
+  ReplaceRelatedRecordsOperation,
+  ReplaceAttributeOperation,
+  RecordIdentity,
+  ReplaceRecordOperation,
+  TransformBuilder,
+} from '@orbit/data';
+import { deepSet, Dict } from '@orbit/utils';
+
+import { JSONAPIOperationsPayload, JSONAPIOperation } from './types';
+import JSONAPISource, { Resource } from '../..';
+
+/**
+ * TODO: import everything below from orbit.js when Operations stuff is merged
+ */
+
+export function transformsToJSONAPIOperations(
```

Comment:
Example usage:
 - within websocket context: https://github.com/sillsdev/appbuilder-portal/pull/504/files#diff-ce742c7207a0c7bb8bd40b7205ef251bR47
 - usage in component-space: https://github.com/sillsdev/appbuilder-portal/pull/504/files#diff-cb43c5b8ff382a226b31e30cc676a443R62